### PR TITLE
Senegal gazettes scrapper 

### PR DESCRIPTION
**Pull request by boswellgathu**
### what does this PR do?
 * Adds opengazettes_sn spider
 * Adds an Item Pipeline

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,101 +1,3 @@
...
-.spyproject
-
-# Rope project settings
-.ropeproject
-
-# mkdocs documentation
-/site
-
-# mypy
-.mypy_cache/
```
Comment (DavidLemayian) [inline .gitignore]: The above items look like they'd be necessary. ☝️

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (boswellgathu) [reply .gitignore]: working to change that

File: opengazettes_sn/items.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#
+# See documentation in:
+# http://doc.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+
+
+class OpengazettesSnItem(scrapy.Item):
+    # define the fields for your item here like:
+    gazette_link = scrapy.Field() # link to the gazette on http://www.jo.gouv.sn/spip.php
```
Comment (maquchizi) [inline opengazettes_sn/items.py]: Avoid inline comments as much as possible. See PEP8 here: https://www.python.org/dev/peps/pep-0008/#inline-comments

**Review by maquchizi — CHANGES_REQUESTED** [review]
Please add all files to be ignored to `.gitignore`. This includes `*.pyc` and `*.egg-info/`

File: opengazettes_sn/items.py (hunk 2 above)
Comment (boswellgathu) [reply opengazettes_sn/items.py]: Sure, this has now been resolved

**Review by DavidLemayian — APPROVED** [review]

**Review by maquchizi — APPROVED** [review]

File: opengazettes_sn/settings.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+SPIDER_MODULES = ['opengazettes_sn.spiders']
+NEWSPIDER_MODULE = 'opengazettes_sn.spiders'
+FILES_STORE_S3_ACL = 'public-read'
+DELTAFETCH_ENABLED = True
+
+# Crawl responsibly by identifying yourself (and your website) on the user-agent
+USER_AGENT = 'opengazettes_sn (+http://www.opengazettes.sn)'
+
+# Obey robots.txt rules
+ROBOTSTXT_OBEY = True
```
Comment (DavidLemayian) [inline opengazettes_sn/settings.py:23]: @GathuBoswell Could we decide not to? 😁

**Review by DavidLemayian — APPROVED** [review]

**Pull request merged**